File: UnityApplication/app/src/main/java/com/example/testUnity/MyUnityPlayerActivity.java
package com.example.testUnity;

import android.os.Bundle;
import android.util.Log;

import com.unity3d.player.UnityPlayerActivity;

public class MyUnityPlayerActivity extends UnityPlayerActivity {
    private static final String TAG = "Player";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        Log.d(TAG, "Running MyUnityPlayerActivity.");
        BackgroundCheckScheduler.scheduleBackgroundCheck(this);
    }
}

File: TestKotlin/app/src/main/java/com/example/testkotlin/MyUnityPlayerActivity.kt
package com.example.testkotlin

import android.os.Bundle
import android.util.Log

import com.unity3d.player.UnityPlayerActivity

class MyUnityPlayerActivity : UnityPlayerActivity() {
    protected override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.d(TAG, "Running MyUnityPlayerActivity.")
        BackgroundCheckScheduler.scheduleBackgroundCheck(this)
    }

    companion object {
        private const val TAG = "Player"
    }
}